server/world/recipes/recipe.go

```diff
@@ -0,0 +1,40 @@
+package recipes
+
+import "github.com/df-mc/dragonfly/server/item"
+
+// Recipe is embedded by all recipe types.
+type Recipe interface {
+	__()
+}
+
+// ShapelessRecipe is a recipe that has no particular shape.
+type ShapelessRecipe struct {
+	recipe
+	// Inputs is a list of items that serve as the input of the shapeless recipe. These items are the items
+	// required to craft the output. The amount of input items must be exactly equal to Width * Height.
+	Inputs []Item
+	// Output is an item that is created as a result of crafting the recipe.
+	Output item.Stack
+	// Priority ...
+	Priority int32
```

Comment:
No point exposing this if we have no use for it.

Reply:
Removed.